airflow/providers/google/cloud/hooks/datafusion.py

```diff
@@ -397,7 +400,13 @@ def start_pipeline(
             "DataPipelineWorkflow",
             "start",
         )
-        response = self._cdap_request(url=url, method="POST")
+
+        if runtime_args != None:
+            args = ast.literal_eval(runtime_args)
+        else:
+            args = None
+
+        response = self._cdap_request(url=url, method="POST", body=args)
```

Comment:
Why pass `runtime_args` as a string? Can't we use `dict`?

Reply:
thanks turbaszek, you are right.
I've updated from string to dict, updated also tests